modules/cache2k/src/main/scala/scalacache/cache2k/Cache2kCache.scala

```diff
@@ -66,12 +61,12 @@ object Cache2kCache {
     * Create a new cache2k cache
     */
   def apply[V](implicit config: CacheConfig): Cache2kCache[V] =
-    apply(new Cache2kBuilder[String, Entry[V]]() {}.build)
+    apply(Cache2kBuilder.of(classOf[String], classOf[Entry[V]]).expireAfterWrite(100000, DAYS).build)
```

Comment:
`Entry` is still needed to avoid runtime error

```scala
  def apply[V](implicit config: CacheConfig): Cache2kCache[V] =
    apply(Cache2kBuilder.of(classOf[String], classOf[V]).expireAfterWrite(100000, DAYS).build)
```
```
java.lang.IllegalArgumentException: The run time type is not available, got: V
  at org.cache2k.configuration.CacheTypeCapture.of(CacheTypeCapture.java:68)
  at org.cache2k.configuration.CacheTypeCapture.of(CacheTypeCapture.java:60)
  at org.cache2k.Cache2kBuilder.<init>(Cache2kBuilder.java:136)
  at scalacache.cache2k.Cache2kCache$$anon$1.<init>(Cache2kCache.scala:64)
  at scalacache.cache2k.Cache2kCache$.apply(Cache2kCache.scala:64)
  ... 36 elided
```